plugins/by-name/vim-suda/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, helpers, ... }:
+
+lib.nixvim.vim-plugin.mkVimPlugin {
+  name = "vim-suda";
+  globalPrefix = "suda#";
+  maintainers = [ lib.maintainers.marcel ];
+
+  settingsOptions = {
+    executable = helpers.defaultNullOpts.mkStr "sudo" ''
+      Path to the sudo binary.
+    '';
+
+    noninteractive = helpers.defaultNullOpts.mkBool false ''
+      If set, suda will not prompt you for a password before saving a file.
+      It is supposed to support a setup with passwordless sudo or doas.
+      Use with care.
+    '';
+
+    prompt = helpers.defaultNullOpts.mkStr "Password: " ''
+      A prompt string used to ask password.
+    '';
+
+    smart_edit = helpers.defaultNullOpts.mkBool false ''
```

Comment:
I think if this setting should just be a `# TODO` until https://github.com/lambdalisue/vim-suda/pull/84 makes it to nixos-unstable?